v0.23.7 release notes (#4174)
voxel51/fiftyone: docs/source/release-notes.rst

.. default-role:: code

FiftyOne Teams 1.5.8
--------------------
*Released March 21, 2024*

Includes all updates from :ref:`FiftyOne 0.23.7 <release-notes-v0.23.7>`.

FiftyOne 0.23.7
---------------
*Released March 21, 2024*

App

- Updated `Have a Team?` link with
  `Book a demo <https://voxel51.com/book-a-demo/?utm_source=FiftyOneApp>`_
  page
- Fixed indexed BooleanFields in Lightning Mode
  `#4139 <https://github.com/voxel51/fiftyone/pull/4139>`_
- Fixed app crash when many None-valued fields exist in the sample modal
  `#4154 <https://github.com/voxel51/fiftyone/pull/4154>`_

Docs

- Added an `Albumentations <https://albumentations.ai/docs/>`_ image augmentation :ref:`integration overview <albumentations-integration>`
  `#4155 <https://github.com/voxel51/fiftyone/pull/4155>`_

> - Adding Places dataset to the zoo

Reviewer: ```suggestion
- Added `Places dataset <http://places2.csail.mit.edu/index.html>`_ to the zoo
```

Reviewer: note this link is currently broken so I don't know what's up but it is the correct link I believe

  `#4130 <https://github.com/voxel51/fiftyone/pull/4130>`_
- Added a Zero-Shot Image Classification Tutorial
  `#4133 <https://github.com/voxel51/fiftyone/pull/4133>`_

> - Added documentation for Google and AWS Cloud Storage :ref:`env vars <cloud-credentials>`

Reviewer: ```suggestion
- Improved documentation for Google and AWS Cloud Storage :ref:`env vars <cloud-credentials>`
```

  `#4151 <https://github.com/voxel51/fiftyone/pull/4151>`_
- Added :ref:`YOLOv8, YOLOv9, and YOLO-World <ultralytics-integration>` to FiftyOne Model Zoo
  `#4153 <https://github.com/voxel51/fiftyone/pull/4153>`_

> 

Reviewer: ```suggestion
- Added `og:image` meta tag to doc pages for better page sharing on socials
  `#4173 <https://github.com/voxel51/fiftyone/pull/4173>`_

```

Plugins and Operators

- Added support for executing operators programmatically in notebooks
  `#4134 <https://github.com/voxel51/fiftyone/pull/4134>`_
- Improved execution of operators during loading of the App
  `#4136 <https://github.com/voxel51/fiftyone/pull/4136>`_
- Added ability to auto-execute operator on opening a dataset
  `#4137 <https://github.com/voxel51/fiftyone/pull/4137>`_
- Improved performance of operator type resolution by only calling :meth:`resolve_input() <fiftyone.core.operator.Operator.resolve_input>` on demand
  `#4152 <https://github.com/voxel51/fiftyone/pull/4152>`_
- Added support for setting saved view in set-view operator
  `#4159 <https://github.com/voxel51/fiftyone/pull/4159>`_
- Added ability to trigger all built-in operators via :class:`ctx.ops <fiftyone.operators.operations.Operations>`
  `#4164 <https://github.com/voxel51/fiftyone/pull/4164>`_
- Fixed issue where JS operator input was not validated when calling `ctx.trigger()` or `executeOperator()` directly
  `#4170 <https://github.com/voxel51/fiftyone/pull/4170>`_

Core

> - Add docs for why wildcard indexes should not be used by default

Reviewer: ```suggestion
- Added docs for why wildcard indexes should not be used by default
```

Reviewer: Move to Docs section above

Reviewer: done

  `#4138 <https://github.com/voxel51/fiftyone/pull/4138>`_
- Improved `super-gradients <https://github.com/Deci-AI/super-gradients>`_ prediction performance with ``model.predict()``
  `#4149 <https://github.com/voxel51/fiftyone/pull/4149>`_
- Passing a :ref:`grouped collection <groups>` to a method that was not specifically designed to handle them now raises better validation errors
  `#4150 <https://github.com/voxel51/fiftyone/pull/4150>`_
- :class:`fiftyone.utils.data.MediaExporter` no longer re-exports media unnecessarily
  `#4143 <https://github.com/voxel51/fiftyone/pull/4143>`_
- Added explicit support for python 3.11 and 3.12
  `#4157 <https://github.com/voxel51/fiftyone/pull/4157>`_
- Added Non-Maximum Suppression utility :meth:`perform_nms() <fiftyone.utils.labels.perform_nms>` to the SDK
  `#4160 <https://github.com/voxel51/fiftyone/pull/4160>`_
- Improved error message when the given dataset name is unavailable
  `#4161 <https://github.com/voxel51/fiftyone/pull/4161>`_

> 

Reviewer: ```suggestion
- Cloning a dataset or view includes custom MongoDB indexes now
  `#4115 <https://github.com/voxel51/fiftyone/pull/4115>`_

```

> 

Reviewer: ```suggestion
- Added ability to change ``map_type`` from the default ``roadmap`` (`carto-positron <https://plotly.com/python/mapbox-layers/>`_) to ``satellite`` (`public USGS map imagery <https://basemap.nationalmap.gov/>`_) in :meth:`location_scatterplot() <fiftyone.core.plots.plotly.location_scatterplot>`
  `#4075 <https://github.com/voxel51/fiftyone/pull/4075>`_

```
![image](https://github.com/voxel51/fiftyone/assets/6363888/84a9008f-edf0-4a82-bd4f-00baae9d1f11)

.. default-role:: code

FiftyOne Teams 1.5.8
--------------------
*Released March 21, 2024*

Includes all updates from :ref:`FiftyOne 0.23.7 <release-notes-v0.23.7>`.

FiftyOne 0.23.7
---------------
*Released March 21, 2024*

App

- Updated `Have a Team?` link with
  `Book a demo <https://voxel51.com/book-a-demo/?utm_source=FiftyOneApp>`_
  page
- Fixed indexed BooleanFields in Lightning Mode
  `#4139 <https://github.com/voxel51/fiftyone/pull/4139>`_
- Fixed app crash when many None-valued fields exist in the sample modal
  `#4154 <https://github.com/voxel51/fiftyone/pull/4154>`_

Docs

- Added an `Albumentations <https://albumentations.ai/docs/>`_ image augmentation :ref:`integration overview <albumentations-integration>`
  `#4155 <https://github.com/voxel51/fiftyone/pull/4155>`_

> - Added `Places2 dataset <http://places2.csail.mit.edu/index.html>`_ to the zoo

Reviewer: ```suggestion
- Added :ref:`Places2 dataset <dataset-zoo-places>` to the zoo
```
